pandas/core/ops/__init__.py

```diff
@@ -672,9 +671,6 @@ def na_op(x, y):
         if is_object_dtype(x.dtype):
             result = comp_method_OBJECT_ARRAY(op, x, y)
 
-        elif is_datetimelike_v_numeric(x, y):
```

Comment:
The removal here is just coupled with some of the other items you've been working on to clean up the datetimelike ops right?

Reply:
Trying to split cleanup changes off from bigger changes in preparation for a non-trivial refactor to fix arithmetic performance.  This check is no longer necessary; I _think_ it was put in place because of older numpy behavior.